Content.Server/Voting/VoteCommands.cs

```diff
@@ -62,13 +70,21 @@ public CompletionResult GetCompletion(IConsoleShell shell, string[] args)
     public sealed class CreateCustomCommand : IConsoleCommand
     {
         [Dependency] private readonly IAdminLogManager _adminLogger = default!;
-
+        [Dependency] private readonly IEntitySystemManager _entitySystem = default!;
+        [Dependency] private readonly IConfigurationManager _cfg = default!;
         private const int MaxArgCount = 10;
 
         public string Command => "customvote";
         public string Description => Loc.GetString("cmd-customvote-desc");
         public string Help => Loc.GetString("cmd-customvote-help");
 
+        // Webhook stuff 
+        private string _webhookUrl = _cfg.GetCVar(CCVars.DiscordVoteWebhook);;
+        private string _serverName = _cfg.GetCVar(CVars.GameHostName);
+        private string _avatarUrl = _cfg.GetCVar(CCVars.DiscordVoteAvatar);
+        private readonly HttpClient _httpClient = new();
+        private string webhookId = string.Empty;
```

Comment:
You do not need to get the webhook's ID, just the link would be enough

Reply:
The ID is used so it can edit the message later, when the vote has finished.